base_wamas_ubl/data/ir_actions_server_data.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
+
+    <record id="ias_get_ubl_version" model="ir.actions.server">
```

Comment:
I don't get why you need server actions... :thinking:

Reply:
because UBL has a lot of templates and we cannot add hard-coding functions, user can add functions that they need for their UBL documents, so I use `ir.actions.server` for this purpose.

it's may not right to use `ir.actions.server` like this, but user can control their document actively